plasmapy/formulary/frequencies.py

```diff
@@ -4,6 +4,7 @@
     "lower_hybrid_frequency",
     "plasma_frequency",
     "upper_hybrid_frequency",
+    "plasma_critical_density",
```

Comment:
The way we are organizing the formulary files now (by dimensions) I guess we should actually move this function to a new file called 'densities.py'. You can just copy structure of this file, e.g. include a __all__ dunder with the name of the function and add it to this modules formulary/__init__.py

Don't worry about aliases or lite functions: that doesn't include what you're doing here.